src/api/modelTypes.js

```diff
@@ -456,3 +456,5 @@ export type Message = $ReadOnly<{
   timestamp: number,
   type: 'stream' | 'private',
 }>;
+
+export type TypingOperation = 'start' | 'stop';
```

Comment:
This doesn't really belong at this spot.

Let's put it in `eventTypes.js`. Give it the name `TypingEventOp`, and put it after `StreamEvent` -- when we later add `TypingEvent` to that file, we'll put it at that spot alphabetically.

Reply:
I have updated this PR, created new method `sendTypingStop`. I don't think in this new version we want to move out type from api, let it be present where it is actually used (used only once). No unnecessary re-directions.